mongoimport/mongoimport_test.go

```diff
@@ -229,7 +229,7 @@ func TestMongoImportValidateSettings(t *testing.T) {
 			So(err, ShouldBeNil)
 			imp.InputOptions.HeaderLine = true
 			imp.InputOptions.Type = CSV
-			imp.IngestOptions.Upsert = true
+			imp.IngestOptions.Mode = "upsert"
```

Comment:
You should use `modeUpsert` here and in the lines bellow. Once they constants are defined, they should be used wherever possible.